releases.yaml

```diff
@@ -1,13 +1,13 @@
 latest:
   slug: LunarLobster
   name: "Lunar Lobster"
-  short_version: "23.04"
-  full_version: "23.04"
+  short_version: "23.10"
+  full_version: "23.10"
   core_version: "22"
   release_date: "April 2023"
-  eol: "January 2024"
+  eol: "July 2024"
   past_eol_date: false
-  release_notes_url: "https://discourse.ubuntu.com/t/lunar-lobster-release-notes/31910"
+  release_notes_url: "https://discourse.ubuntu.com/t/mantic-minotaur-release-notes/35534"
 lts:
```

Comment:
You don't need any of these changes, we still haven't released 23.10, this will change on October 12

Reply:
The copy doc had those changes

Reply:
Liza, you need to point your PR to `23-10-release` branch instead of `main`. Then you will see that these changes are already applied there.